config/settings.yml

```diff
@@ -5128,6 +5128,8 @@ callsigns:
   - Warrior
   - Wizzard
   - Zapper
+  - Dustoff
+  - Medevac
```

Comment:
Dustoff is an established callsign for CASEVAC units, but Medevac is not a callsign alone. It is just a term which a civilian air ambulance should add to its flight number when talking over the radio to make sure that they get priority due to their mission. It's not used in the military. The callsign list here is for tactical callsigns.